x-pack/plugins/timelines/common/types/timeline/actions/index.ts

```diff
@@ -118,10 +118,12 @@ export interface BulkActionsObjectProp {
 }
 export type BulkActionsProp = boolean | BulkActionsObjectProp;
 
+export type AlertWorkflowStatus = 'open' | 'closed' | 'acknowledged';
+
 /**
  * @deprecated
  * TODO: remove when `acknowledged` migrations are finished
  */
 export type InProgressStatus = 'in-progress';
 
-export type AlertStatus = 'open' | 'closed' | 'acknowledged' | InProgressStatus;
+export type AlertStatus = AlertWorkflowStatus | InProgressStatus;
```

Comment:
I am wondering why the `in-progress` status is still needed here. Shouldn't it be removed from security solutions too? @dplumlee

Reply:
I've kept it since it was in `master`. I can remove it if necessary